Started by upstream project "[8mha:////4HpW8/yC0/FnkCkdsV0PRVqyrFy/71TMAPE9KGkyLfEXAAAAlR+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzWEgZu/az8JP3cShAJAFU3furAAAAA[0mmyjob" build number [8mha:////4PQ7ldKUxviwzw7PrMmeIJC2snukxvVvqW/5KBCLHyZAAAAAlR+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLQIwSBh79rPwk/dxKEGkGAIfrEa/BAAAA[0m6
originally caused by:
 Started by user [8mha:////4MHfU9xnJk0TgBN1NRm7l/3mu30lymz2z/ypgvOlQqffAAAAlx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyWEgZu/dLi1CL9xJTczDwAZ3pnlMAAAAA=[0mjani
Running as SYSTEM
Building remotely on [8mha:////4L8zriXMW/yHHh1KChGlzyLZbpSGe6BrTMgiX08DlqPDAAAAmx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAy2EgYB/eT83ILSktQi/dzKPKBLASShNV3FAAAA[0mmynode (mylabel) in workspace /home/jenkins/workspace/myjob1
[myjob1] $ /bin/sh -xe /tmp/jenkins6109107044043458975.sh
+ sudo docker run --name jc2 -d -p 8090:8080 jani1502/hello:6
docker: Error response from daemon: Conflict. The container name "/jc2" is already in use by container "3af7bbc7ecb8f05a943e364991ad8d286ee002d937f59fd9ac3e7a04002a69f2". You have to remove (or rename) that container to be able to reuse that name.
See 'docker run --help'.
Build step 'Execute shell' marked build as failure
Finished: FAILURE
